### PR TITLE
Feature/nav update

### PR DESCRIPTION
**Pull request by xtine**
Update the nav on eregs to latest version.

screen shot 2017-04-12 at 11 51 51 pm

### CODE REVIEW

File: fec_eregs/templates/regulations/main-header.html [hunk 1]
```diff
@@ -10,33 +10,42 @@
   <div class="masthead">
     <a href="{{ FEC_CMS_URL }}/" title="Home" class="site-title"><span class="u-visually-hidden">Federal Election Commission | United States of America</span></a>
     <ul class="utility-nav list--flat">
-      <li class="utility-nav__item is-disabled">About</li>
+      <li class="utility-nav__item"><a href="{{ FEC_CMS_URL }}/about/">About</a></li>
```
Comment (noahmanger) [inline fec_eregs/templates/regulations/main-header.html]: This should actually be removed. We dont' include it in the other headers.

File: fec_eregs/templates/regulations/main-header.html [hunk 2]
```diff
@@ -10,33 +10,42 @@
...
     </ul>
   </div>
   <nav class="site-nav js-site-nav">
     <div id="site-menu" class="site-nav__container">
       <ul class="site-nav__panel site-nav__panel--main">
-        <li class="site-nav__item site-nav__item--with-dropdown" data-submenu="data">
-          <a href="{{ FEC_WEB_URL }}/" class="site-nav__link">Campaign finance data</a>
+        <li><h2 class="site-nav__title u-under-lg-only">Menu</h2></li>
+        <li class="site-nav__item u-under-lg-only">
+          <a class="site-nav__link" href="{{ cms_url }}" rel="home">
```
Comment (noahmanger) [inline fec_eregs/templates/regulations/main-header.html]: Looks like these variables are different (`FEC_CMS_URL` vs `cms_url`) here.

File: fec_eregs/templates/regulations/main-header.html [hunk 3]
```diff
@@ -10,33 +10,42 @@
...
+            <span class="site-nav__link__title">Home</span>
+          </a>
         </li>
-        <li class="site-nav__item">
-          <a href="{{ FEC_CMS_URL }}/registration-and-reporting/" class="site-nav__link">Registration and reporting</a>
+        <li class="site-nav__item" data-submenu="data">
+          <a href="{{ FEC_WEB_URL }}" class="site-nav__link {% if parent == 'data' %}is-parent{% endif %}">
```
Comment (noahmanger) [inline fec_eregs/templates/regulations/main-header.html]: You can remove the `if parent` checks and just add `is-parent` to legal resources.

File: fec_eregs/templates/regulations/main-header.html [hunk 4]
```diff
@@ -10,33 +10,42 @@
...
-        <li class="site-nav__item utility-nav__item">
-          <a href="{{ FEC_CMS_URL }}/calendar/" class="site-nav__link">Calendar</a>
+        <li class="site-nav__item utility-nav__item u-under-lg-only">
+          <a class="site-nav__link" href="{{ FEC_CMS_URL }}/calendar">Calendar</a>
         </li>
       </ul>
     </div>
-    <button for="nav-toggle" class="js-nav-toggle site-nav__button site-nav__button--left" aria-controls="site-menu">Menu</button>
-    <a title="Home" href="{{ FEC_CMS_URL }}/" class="site-title"><span class="u-visually-hidden">Federal Election Commission | United States of America</span></a>
+    <button for="nav-toggle" class="js-nav-toggle site-nav__button" aria-controls="nav-main">Menu</button>
```
Comment (noahmanger) [inline fec_eregs/templates/regulations/main-header.html]: Should this be left `site-menu`? That's the `id` above.

**Comment by xtine** [comment]
@noahmanger: Updated, thanks for catching the things I missed.

**Comment by jenniferthibault** [comment]
Noah reviewed code & Christine & I paired with it running locally to make sure it's working right. 👍  👍

**Pull request closed without merging**